website/src/releases/2020-10/release-v17.mdx

```diff
@@ -0,0 +1,233 @@
+
+mc-scripts serve
+```
+
+## Removing the `mc-http-server` package
+
+The `@commercetools-frontend/mc-http-server` package has been deprecated and will not receive any updates.
+
+With the usage of the `compile-html` command there is no need to have a pre-configured HTTP server anymore. If you are using this package, we recommend to use any other HTTP server package to serve your static files.
+
+<Info>
+
+Remember that after building your production bundles you need to [compile the Custom Application](/deployment/compiling-a-custom-application#deployment) for production usage.
+
+</Info>
+
+## New command `serve`
+
+We added a new command `mc-scripts serve` that can be used to start your Custom Application **locally** in `production` mode after it has been compiled.
+
+The command starts an HTTP server to serve the static assets from the `public` folder.
+
+<Warning>
+
+This command should only be used locally to test the Custom Application in production mode, as it contains the development routes for `/login` and `/logout`. Do not use this command to serve your Custom Application in production.
+
+</Warning>
+
+Before:
+
+```json title="package.json"
+{
+  "start:prod:local": "NODE_ENV=production MC_APP_ENV=development dotenv -- mc-http-server --use-local-assets"
+}
+```
+
+After:
+
+```json title="package.json"
+{
+  "compile-html:local": "NODE_ENV=production MC_APP_ENV=development dotenv -- mc-scripts compile-html --transformer @commercetools-frontend/mc-dev-authentication/transformer-local.js",
+  "start:prod:local": "yarn compile-html:local && mc-scripts serve",
+}
+```
+
+## Removing the `extract-intl` command
+
+The `mc-scripts extract-intl` command has been removed in favor of the [official `@formatjs/cli` package](https://formatjs.io/docs/tooling/cli#extraction).
+
+We recommend to update your script to extract Intl messages to use the `formatjs extract` command.
+
+Before:
+
+```json title="package.json"
+{
+  "i18n:build": "mc-scripts extract-intl --output-path=$(pwd)/src/i18n/data 'src/**/!(*.spec).js' --build-translations"
+}
+```
+
+After:
+
+```json title="package.json"
+{
+  "extract-intl": "formatjs extract --format=$(pwd)/intl-formatter.js --out-file=$(pwd)/src/i18n/data/core.json 'src/**/!(*.spec).js'"
+}
+```
+
+where the `intl-formatter.js` should be defined as something like this, depending on your translation tool's needs:
+
+```js title="intl-formatter.js"
+// https://formatjs.io/docs/tooling/cli#extraction
+exports.format = function format(extractedMessages) {
+  return (
+    Object.keys(extractedMessages)
+      // transform strings to lowercase to imitate phraseapp sorting
+      .sort((a, b) => a.toLowerCase().localeCompare(b.toLowerCase()))
+      .reduce(
+        (allMessages, messageId) => ({
+          ...allMessages,
+          // Return a simple key/value JSON object.
+          [messageId]: extractedMessages[messageId].defaultMessage,
+        }),
+        {}
+      )
+  );
+};
+```
```

Comment:
I don't think it is necessary to add this examples.
it would be enough and suffice to link to the message extraction of `formatjs`
https://formatjs.io/docs/getting-started/message-extraction

Reply:
This file is what we have also included in the starter app and the playground app. I think it wouldn't hurt to have it documented here as well.

Reply:
very well.